proto/beacon/rpc/v1/services.proto

```diff
@@ -196,3 +197,8 @@ enum ValidatorStatus {
   EXITED = 5;
   EXITED_SLASHED = 6;
 }
+
+message TreeBlockSlotRequest {
+  uint64 slot_number_from = 1 ;
```

Comment:
slot implies number here, I like `slot_from` and `slot_to` better

Reply:
Done.